cli/src/pcluster/validators/ec2_validators.py

```diff
@@ -102,12 +102,20 @@ def _validate(self, key_name: str):
 class PlacementGroupIdValidator(Validator):  # TODO: add tests
     """Placement group id validator."""
 
-    def _validate(self, placement_group_id: str):
-        if placement_group_id:
-            try:
-                AWSApi.instance().ec2.describe_placement_group(placement_group_id)
-            except AWSClientError as e:
-                self._add_failure(str(e), FailureLevel.ERROR)
+    def _validate(self, placement_group):
+        if placement_group.id:
+            if not placement_group.is_implied("enabled") and not placement_group.enabled:
+                self._add_failure(
+                    "PlacementGroup Id can not be set when setting 'Enabled: false' in compute "
+                    "resource's configuration section. Please remove PlacementGroup Id if you don't want"
```

Comment:
NIT: The placement group is in the Queue's [Networking Section](https://docs.aws.amazon.com/parallelcluster/latest/ug/Scheduling-v3.html#Scheduling-v3-SlurmQueues-Networking).

Reply:
Done